src/managers/GuildScheduledEventManager.js

```diff
@@ -68,7 +68,11 @@ class GuildScheduledEventManager extends CachedManager {
         name,
         privacy_level: privacyLevel,
         scheduled_start_time: new Date(scheduledStartTime).toISOString(),
-        scheduled_end_time: scheduledEndTime ? new Date(scheduledEndTime).toISOString() : undefined,
+        scheduled_end_time: scheduledEndTime
+          ? new Date(scheduledEndTime).toISOString()
+          : scheduledEndTime === null
+          ? scheduledEndTime
+          : undefined,
```

Comment:
any reason you don't just.. set it to scheduledEndTime instead of doing these checks? Everything truthy will go down the new Date path, might as well simplify it

Reply:
I don't know why I did it like that ^._.^

![genius](https://user-images.githubusercontent.com/51517103/143262349-70b8a12e-107c-4a1b-93fe-f6b9221927ad.jpg)